mod.ts

```diff
@@ -501,7 +657,10 @@ export type HookEvent =
   | EventIdentityEmailVerified
   | EventIdentityPhoneVerified
   | EventIdentityEmailUnverified
-  | EventIdentityPhoneUnverified;
+  | EventIdentityPhoneUnverified
```

Comment:
Looks like `HookEvent` is only for blocking events. Maybe we should remove blocking events from here, and rename it as `NonBlockingHookEvent`?

This is because it is impossible to handle blocking event + non-blocking event in the same hook file.